ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -20,6 +20,11 @@ class TabBarItemView: UIControl, TokenizedControlInternal {
     override var isEnabled: Bool {
         didSet {
             isUserInteractionEnabled = isEnabled
+            if isEnabled {
```

Comment:
Should we be checking if the value change before changing the accessibilityTraits?